templates/engage/stuckstack-ebook.html

```diff
@@ -0,0 +1,31 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}How to escape StuckStack and profit from your OpenStack investment{% endblock %}
+
+{% block meta_description %}This eBook offers a four-point plan to remobilising the StuckStack infrastructure and regaining organisational agility.{% endblock %}
+
+{% block meta_copydoc %}https://app-sjg.marketo.com/#LP5282A1LA1{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="How to escape StuckStack and profit from your OpenStack investment" image="410edf8d-cost_%28piggy-bank%29.png" url="#the-form" cta="Download the eBook" %}
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+        <p>When OpenStack works well, it delivers on its immense promise. However, any software comprised of many different components, third party integrations and revisions needs tight version control. If not, complexity can become its downfall. When an OpenStack deployment falls too far behind the pace of upgrades, it becomes impossible to keep up with the latest versions. We have called this condition StuckStack and
```

Comment:
please use 2 spaces indentation instead of 4 as this is a Canonical best practice https://canonical-web-and-design.github.io/practices/coding/html.html